### PR TITLE
Add bot configs to enable bot features

### PR DESCRIPTION
**Pull request by anthonydresser**
What does this add?

- Auto classifying new issues into bug/feature and into different areas
- Auto closing and commenting of issues that are labeled with `duplicate`
- Auto lock issues that have been closed for more than 45 days and have not had an update in 3 days (configurable numbers)
 ** WARNING: THIS WILL CAUSE A LOT OF ISSUES TO BE LOCKED **
- Auto close issues that have the tag `more info` and have not had a community comment (non repo collaborator) in more than 7 days (configurable)
 ** WARNING THE FOLLOWING ISSUES WILL BE CLOSED BY THIS **
- #1562 @alanrenmsft 
- #1720 @kevcunnane 
- #2011 @kburtram 
- #2310 @kburtram 
- #2741 @kburtram 
- #3046 @kburtram 
- #3468 @kburtram 
- #3469 @kburtram 
- #3815 @kburtram @yualan 
- #3904 @kburtram 
- #4027 @MattIrv 

- Comment on new issues if potential duplicates are found.

### CODE REVIEW

**Comment by anthonydresser** [comment]
For people interested in what it looks like https://github.com/anthonydresser/testissues/issues

**Comment by anthonydresser** [comment]
The issues that will be closed by the bot I have tagged the person who added the `more info` label. Please verify you are ok with that issue being closed.

**Comment by alanrenmsft** [comment]
@anthonydresser looks good to me to close it.

**Comment by MattIrv** [comment]
mine is good to close

File: .github/commands.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	perform: true,
+	commands: [
+        {
+            type: 'label',
+            name: 'duplicate',
+            allowTriggerByBot: true,
+            action: 'close',
+            comment: "Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for existing issues [here](https://aka.ms/vscodeissuesearch). See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
```
Comment (MattIrv) [inline .github/commands.yml]: the links in this message need to be updated

File: .github/needs_more_info.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    daysUntilClose: 7,
+    needsMoreInfoLabel: 'more info',
+    perform: true,
+    closeComment: "This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
```
Comment (MattIrv) [inline .github/needs_more_info.yml]: this link also needs to be updated

File: .github/commands.yml (hunk 1 above)
Comment (anthonydresser) [reply .github/commands.yml]: Yep, @yualan is working on that now.

**Comment by anthonydresser** [comment]
I'm actually going to turn `perform: false` for all the features and monitor it in the near term to double ensure nothing goes wrong. I'll make another pr making `perform: true` for everything once I see there are no problems.

**Comment by anthonydresser** [comment]
Going to merge this in with bot actions disabled so I can monitor logs. Once @yualan gets me updated comment strings I'll update them in the configs and enable those features.

**Pull request merged**